static/src/javascripts/projects/commercial/modules/dfp/get-advert-by-id.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+import dfpEnv from 'commercial/modules/dfp/dfp-env';
+
+const getAdvertById = (id: string): ?Object =>
+    id in dfpEnv.advertIds ? dfpEnv.adverts[dfpEnv.advertIds[id]] : null;
+
+export { getAdvertById };
```

Comment:
maybe next time prefer `export default`, until all the files are ported, then we can pass through and remove the default exports? Saves updating a number of legacy references once, and then having to put the refs back when the legacy file is converted.